invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/ItemDetails/components/ItemPendingLoans/ItemPendingLoans.js

```diff
@@ -1,180 +1,63 @@
     const { document_pid, item_pid } = this.props.item.metadata;
-    const qs = loanApi.buildLoansQuery(document_pid, item_pid, null, null);
-    const url = `${BackOfficeURLS.loansSearch}?q=${qs}`;
-    this.props.history.push(url);
-  };
-
-  _renderFooter = pendingLoans => {
-    const total = pendingLoans.length;
-    return total > this.props.showMaxPendingLoans ? (
-      <Table.Footer fullWidth data-test="footer">
-        <Table.Row>
-          <Table.HeaderCell colSpan="5" textAlign="right">
-            <span>
-              Showing first {this.props.showMaxPendingLoans} pending requests
-            </span>
-            <Button size="small" onClick={this._handleShowAllClick}>
-              Show all
-            </Button>
-          </Table.HeaderCell>
-        </Table.Row>
-      </Table.Footer>
-    ) : null;
-  };
-
-  _handleSortClick = sortByValue => {
-    const { document_pid, item_pid } = this.props.item.metadata;
-    if (this.props.currentSortBy === sortByValue) {
-      // change sort order
-      this.pendingLoansChangeSortOrder(document_pid, item_pid);
-    } else {
-      // change sort by
-      this.pendingLoansChangeSortBy(document_pid, item_pid);
-    }
-  };
-
-  _renderPendingLoans = pendingLoans => {
-    const _pendingLoans = pendingLoans
-      .slice(0, this.props.showMaxPendingLoans)
-      .map(loan => this._renderPendingLoan(loan));
-    return (
-      <Table singleLine selectable className="item-pending-loans">
-        <Table.Header>
-          <Table.Row data-test="header">
-            <Table.HeaderCell />
-            <Table.HeaderCell textAlign="center">Loan</Table.HeaderCell>
-            <Table.HeaderCell textAlign="center">Patron</Table.HeaderCell>
-            <Table.HeaderCell>
-              <span>Request created</span>
-              <Popup
-                trigger={
-                  <Icon
-                    name="sort"
-                    link
-                    onClick={() => this._handleSortClick('transaction_date')}
-                  />
-                }
-                content="Sort by request date asc/desc"
-                size="mini"
-                position="top center"
-              />
-            </Table.HeaderCell>
-            <Table.HeaderCell>
-              <span>Period of interest</span>
-              <Popup
-                trigger={
-                  <Icon
-                    name="sort"
-                    link
-                    onClick={() => this._handleSortClick('start_date')}
-                  />
-                }
-                content="Sort by period of interest asc/desc"
-                size="mini"
-                position="top center"
-              />
-            </Table.HeaderCell>
-          </Table.Row>
-        </Table.Header>
-        <Table.Body>{_pendingLoans}</Table.Body>
-        {this._renderFooter(pendingLoans)}
-      </Table>
+    this.props.history.push(
+      this.showAllUrl(document_pid, item_pid, 'PENDING', null)
```

Comment:
I would have generated the path here probably, it belongs to the UI layer.
If you need to re-use it, let's find a common place around URL handling.